### ISSUE TITLE
Replace slack link with discord link on contact bar at top of page. 

### ISSUE THREAD

**Issue by brandon-powell** [devnetmsu/devnet-website#20]

**Comment by nakleiderer** [comment]
Good idea. Has some implications though. Slack had open enrollment options with email addresses. I don't think discord does. I think we'd have to provide an invite link. Which means we'd need a user role that can only see a few channels, then open it up once we know the user is trusted. @evandixon: thoughts?
